### DISCUSSION TITLE
How to access and modify config.php?

### DISCUSSION THREAD

**Discussion by naveennamani** [nextcloud/all-in-one#1187] [Questions] [answered]
I've downloaded the aio docker image and running it in windows machine as per the commands given in the README.
However, my PC is in an environment where internet is not available.
So, I'm getting some errors asking to fix dns settings.

After searching through the issues, I found that I need to set "is_ethernet_connected" to false in configuration.php file.
However, I'm unable to find just how to do that. There were some instructions in README about editing configuration.php, but as I'm
using windows OS and running a linux container I'm unable to access the files in the container.

Can I somehow create a new image file using Dockerfile as follows

```
FROM nextcloud/all-in-one:latest

COPY /path/to/local/configuration.php /path/to/container/configuration.php  # <-- I don't know which path this is
```

If the above approach works, where shall I copy my local configuration.php file?

Thanks.

**Comment by szaimen** [comment, marked as answer]
Just change the config file inside the container with docker exec...

**Reply by naveennamani** [reply]
I'm just lost there finding the location of the config file inside the container. Can you please elaborate a little?

**Reply by szaimen** [reply]
It is stored here inside the container: /mnt/docker-aio-config/data/configuration.json

**Comment by szaimen** [comment]
However I fear AIO doesnt support such an evironment. (Without internet access).

Windows itself is supported but you'll definitely need internet access.

**Reply by naveennamani** [reply]
Can you please tell me a solution in that case, can I still host it without docker?

**Reply by szaimen** [reply]
Yes, definitely!
